RELEASES.md

```diff
@@ -10,27 +10,24 @@ After review of a protocol-branch and before each release the following steps wi
 
         2.  `protocolhelper::check_structure()`
 
-    2.  Check any error messages resulting from these checks and address them.
-        Repeat this step until all checks succeed.
-        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
-        Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
-        If this commit is added in the remote, pull it into your local clone.
+    Check any error messages resulting from these checks and address them.
+    Repeat this step until all checks succeed.
+    In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
+    Note that during this automatic run, a commit to update the version number may be automatically pushed to the remote.
+    If this commit is added in the remote, pull it into your local clone.
 
-2.  A second GitHub Action will be run when a pull request to be merged into the main branch *is approved*.
-
-    1.  This action will automatically update:
+    3.  This action will also automatically update:
 
         1.  the repo-wide `NEWS.md` file with `protocolhelper:::update_news_release()`
 
         2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-    1.
```

Comment:
Comment on line below this one: this is not the second Github Action, it is incorporated in the first one.  So check the lines below as well.  Or you may want to move up the actions above this indicated line (lines 19-24) and put them just below the other steps of the GHA (line 12).  After this, you just need one of the texts describing what to do (either lines 13-17 or lines 25-28), or you can mix them together and keep the most relevant parts.

Reply:
Good catch. Fixed in [e927edb](https://github.com/inbo/protocolsource/pull/73/commits/e927edb052968dd3fe13981295049b92693b48d9)